sql/core/src/test/scala/org/apache/spark/sql/connector/SimpleWritableDataSource.scala

```diff
@@ -134,7 +129,7 @@ class SimpleWritableDataSource extends SimpleTableProvider with SessionConfigSup
     private val path = options.get("path")
     private val conf = SparkContext.getActive.get.hadoopConfiguration
 
-    override def schema(): StructType = tableSchema
+    override def schema(): StructType = TestingV2Source.schema
```

Comment:
ditto, `SimpleBatchTable.schema` is already `TestingV2Source.schema`

Reply:
yes, it‘s unnecessary.